instances/widgets.treasury-factory.near/widget/pages/settings/system-updates/UpdateRegistry.jsx

```diff
@@ -1,11 +1,11 @@
 return [
-  /*{
-    id: 1,
+  {
```

Comment:
Please revert changes in this file, as we do not want to publish any update that the users should pay attention to. The commented entry here should just be an example